parquet-tools/src/main/java/parquet/tools/command/SizeCommand.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2013 ARRIS, Inc.
```

Comment:
@rdblue I had used header in other classes in parquet-tool. I have changed headers in my classes. Do you want to fix all classes to use Apache copyright also? It is my first open source commit so many common mistakes, won't repeat !!